modules/images-configuration-create-config-map.adoc

```diff
@@ -0,0 +1,60 @@
+id="steps-for-manual-configmap-creation"]
+= Steps create the image signature configmap
+
+The following commands will create and apply the image signature configmap to the cluster you want to update. 
+
+.Procedure
+
+. Find the link:https://access.redhat.com/solutions/4583231[version] you want the cluster to update.
+
+. Add the version to OCP_RELEASE_NUMBER environment variable.
++
+----
+$ export OCP_RELEASE_NUMBER=<{product-version}.0>
```

Comment:
[Still][1] no need to `export` this or other other variables you're using.  Although I see #22008 is still open, so you're still free to punt ;).  Also in this space is @smarterclayton's pushback [here][2] (which I attempted to rebut in my follow-up comment in that thread).

[1]: https://github.com/openshift/openshift-docs/pull/21993#discussion_r423298514
[2]: https://github.com/openshift/openshift-docs/pull/21993#discussion_r424620899